### PR TITLE
Fix pages structure

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

*See #2 for current behavior*

**What is the new behavior (if this is a feature change)?**
- The folder- and filestructure of components/pages has been fixed to maintain the standard

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 @import 'utilities/base';
 
 // Local
```
Comment (kleberbaum) [inline src/index.scss]: NOTE: I know this comment is not new, but It's kinda more a headline than a comment.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
It should be k to merge, but I don't like the Local comment.

File: src/index.scss (hunk 1 above)
Comment (Aichnerc) [reply src/index.scss]: Thank you for your notice. This issue has been taken care of.

File: src/index.scss [hunk 2]
```diff
@@ -1,10 +1,10 @@
 // Utilities
```
Comment (kleberbaum) [inline src/index.scss]: NOTE: This comment is also more a headline than a comment.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
It would be ok to merge, but I found another comment I don't like.

**Review by kleberbaum — APPROVED** [review]
Is ready to merge. ^^

**Pull request merged**